### PR TITLE
updated spinner.md

### PR DESCRIPTION
**Pull request by lakshita10341**
i have done the changes. please verify it

### CODE REVIEW

File: docs/docs/widgets/spinner.md [hunk 1]
```diff
@@ -4,53 +4,55 @@ title: Spinner
...
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Layout
 
 | <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `fx` to set the value `{{true}}` or `{{false}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/spinner.md:42]: Hey @lakshita10341, the fx should be in bold and not enclosed in back tick, can you please make this change in the whole doc and both versions where ever applicable.

File: docs/docs/widgets/spinner.md [hunk 2]
```diff
@@ -4,53 +4,55 @@ title: Spinner
...
+| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `fx` to set the value `{{true}}` or `{{false}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/spinner.md:43]: Can you please also add a full stop at the end of sentence here? Please do this for both the sentence in the table and in both versions.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

**Comment by PriteshKiri** [comment]
Hey @lakshita10341,

Can you please make the changes asked by @RudraPratik30 ?

**Comment by RudraPratik30** [comment]
Hey @lakshita10341!
Closing this PR due to lack of progress.

**Comment by lakshita10341** [comment]
Hi @PriteshKiri 
I have done the changes which was said earlier and the PR was merged. Sorry for not replying on above suggested changes because I was busy that time somewhere else. So didn't see the mail. But now it is showing my pr is not merged which was merged earlier.
Can this issue be resolved?

**Pull request closed without merging**